presto-main/src/main/java/io/prestosql/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -402,12 +403,19 @@ public PlanNode visitJoin(JoinNode node, RewriteContext<Expression> context)
 
             switch (node.getType()) {
                 case INNER:
+                    ImmutableList.Builder<Expression> expressionListBuilder = ImmutableList.builder();
+                    expressionListBuilder.add(joinPredicate);
+                    node.getCriteria().stream()
+                            .map(criteria -> new IsNotNullPredicate(criteria.getLeft().toSymbolReference()))
```

Comment:
add also `criteria.getRight()`

Reply:
But for right side table it will be added when we process the Inner join (with equi join condition).

Reply:
I think explicitly creating conditions for both sides would be cleaner.

Reply:
Have added for both side